nservicebus/recoverability/index.md

```diff
@@ -39,6 +39,8 @@ By default up to five immediate retries are performed if the message processing
 
 Note: The configured value describes the minimum number of times a message will be retried if its processing consistently fails. Especially in environments with competing consumers on the same queue, there is an increased chance of retrying a failing message more times across the endpoints.
 
+Note: Depending on the concurrency and transactionallity settings of the endpoint immediate retries might happen even if configured to be off. To ensure that no retries is performed the endpoints needs to be configured to be either [non transactional](/nservicebus/transports/transactions.md) or the [concurrency retricted to 1](/nservicebus/operations/tuning.md).
```

Comment:
> can you give an example of when immediate retries might happen?

It will happen if running v6 with MSMQ or SqlServer transports and a exception happens during transaction scope dispose/commit, msmq transaction dispose, sql connection dispose etc.

>So, are you saying that retries will happen even if it is turned off? 

Yes

>In that case is this is a bug?

It could be argued that it is yes but the decision was to start by documenting this since there is a workaround? (turn transactions off)